### PR TITLE
update README-zh_CN.md

### PR DESCRIPTION
**Pull request by HashCookie**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ X] Site / documentation update

### 🔗 Related issue link
https://github.com/ant-design/ant-design/blob/master/README-zh_CN.md

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ x]  Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 67b6f21</samp>

This pull request improves the Chinese README file for the ant-design project. It updates the badges, sections, and links to match the current state and functionality of the project. It also enhances the document's readability and usefulness for potential users and contributors.

### <samp>🤖 Generated by Copilot at 67b6f21</samp>

* Add a badge for Argos Visual Testing to show the status of the visual testing and link to the Argos dashboard ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L7-R17))
* Update the badge name for issues-helper to reflect the new GitHub organization name of actions-cool ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L37-R46))
* Reformat the table of browser compatibility to improve the readability and alignment ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L66-R71))
* Fix a syntax error that caused the code block for installation commands to be rendered incorrectly ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L73-L75))
* Update the section "示例" to remove the outdated reference to the "定制主题" section and add a new section about "TypeScript" ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L93-R100))
* Simplify the section "链接" to remove the redundant and outdated links ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L121-R122))
* Revise the section "参与共建" to provide more guidance and information for potential contributors ([link](https://github.com/ant-design/ant-design/pull/44339/files?diff=unified&w=0#diff-099beda0196f3588b806c1d7ace4d054b6a1caf09bfede3b0c1a9ac7cfb74473L145-R155))

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -4,15 +4,17 @@
   </a>
 </p>
 
+
 <h1 align="center">Ant Design</h1>
 
 <div align="center">
 
+
```
Comment (afc163) [inline README-zh_CN.md:12]: Don't add this.

File: README-zh_CN.md [hunk 2]
```diff
@@ -4,15 +4,17 @@
   </a>
 </p>
 
+
```
Comment (afc163) [inline README-zh_CN.md:7]: Don't add this.

File: README-zh_CN.md [hunk 3]
```diff
@@ -63,16 +67,13 @@
...
-| --- | --- | --- | --- | --- |
-| Edge | last 2 versions | last 2 versions | last 2 versions | last 2 versions |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Edge                                                         | last 2 versions                                              | last 2 versions                                              | last 2 versions                                              | last 2 versions                                              |
```
Comment (afc163) [inline README-zh_CN.md:71]: Should not update this.

**Review by AbdullahWins — APPROVED** [review]
good job!

**Pull request closed without merging**